ecommerce/src/App.css

```diff
@@ -0,0 +1,496 @@
+:root {
+  --font-primary: "Nunito Sans", sans-serif;
+  --font-seconday: "Montserrat", sans-serif;
+  --gradient: linear-gradient(
+    90deg,
+    rgba(255, 92, 94, 1) 0%,
+    rgba(245, 97, 191, 1) 100%
+  );
+  --gradient-bg: rgb(255, 92, 94);
+}
+
+.App {
+  background: #f2f2f2;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+.Hero {
+  min-height: 90vh;
+}
+
+.featured {
+  font-family: "montserrat" serrif;
+  font-size: 3.25rem;
+  display: flex;
+  font-family: "nunito", "serrif";
+  margin-top: 1rem;
```

Comment:
If you have created css variables for fonts, I would like to suggest you to use it instead of hard coding at line no. 25 and 28
and also spelling of serif is not correct